src/Makefile.linux

```diff
@@ -14,7 +14,7 @@ LDFLAGS = -shared -Wl,-soname,libSDL2-2.0.so.0
 LDFLAGS+= -Wl,--no-undefined
 LDLIBS  = -ldl
 
-SHLIB = libSDL2-2.0.so.0.2900.0
+SHLIB = libSDL2-2.0.so.0.2800.0
```

Comment:
I don't think this is right. What is the output filename when built with CMake?

Reply:
I would expect that for 2.28.50, it should be `libSDL2-2.0.so.0.2800.50`.

Reply:
Certainly with my packager hat on, I'm always going to prefer a conventional build system (Meson, CMake, Autotools) over a hand-crafted Makefile. I haven't packaged sdl2-compat for Debian experimental yet, but I used CMake for sdl12-compat and SDL3.

Reply:
This is fixed now. I forgot how complex the numbering scheme was and pushed a bit too early.

I think these files are inherited from sdl12-compat.
I'll remove them.

Reply:
Makefile.* is Ozkan's thing, I don't think it's meant for use by the general public.

Reply:
> Yeah, I'm thinking we don't need Makefile.linux or Makefile.darwin. Why were these added?

So that I won't have to deal with cmake crapola everytime. They make life easier for me.